src/module/settings.js

```diff
@@ -73,6 +73,15 @@ export function registerSystemSettings() {
     default: false,
   });
 
+  game.settings.register(MODULE_ID, SETTINGS_KEYS.AUTO_SELECT_BEST_CARD, {
+    name: 'SETTINGS.AutoSelectBestCard',
+    hint: 'SETTINGS.AutoSelectBestCardHint',
+    scope: 'world',
+    config: true,
+    type: Boolean,
+    default: false,
+  });
+
```

Comment:
I realize I have completely misunderstood the intention with the "Automatically Draw Initiative" setting. Only when working with the drawing rules did it dawn on me. 

Seeing that the wording is bad I elected to create a new setting that conveys the intention of the one above and replaces it. A migration could be done, but I think I am not the only one who has misunderstood the intention of said setting (and the effect of not migrating is fairly harmless).